spec/support/vcr.rb

```diff
@@ -61,4 +62,12 @@
       i.send(env).headers.update('Authorization' => 'Bearer <TOKEN>')
     end
   end
+
+  c.register_request_matcher :uri_ignoring_path_parameters do |request1, request2|
```

Comment:
We actually might want to name this something more specific to the `isValidSMUser` route like `sm_user_ignoring_path_param` or whatever you think, but otherwise looks good

Reply:
I like it! I'll go with that.